noether_examples/src/mesh_filtering_client.cpp

```diff
@@ -104,12 +104,14 @@ class MeshFilteringClient
     goal.filter_group = filter_group;
     goal.surface_meshes.push_back(mesh_msg);
     ac_.sendGoal(goal);
+    ros::Time start_time = ros::Time::now();
     if(!ac_.waitForResult(ros::Duration(GOAL_WAIT_PERIOD)))
     {
       ROS_ERROR("Failed to filter mesh");
       return false;
     }
-    ROS_INFO("Got filtered results");
+    ros::Duration time_elapsed = ros::Time::now() - start_time;
+    ROS_INFO("Got filtered results, process took %f seconds",time_elapsed.toSec());
```

Comment:
Would it be worth adding these to the filter group/manager base class instead of just the mesh filtering client?

Reply:
Add the whole MeshFilteringClient class?

Reply:
No, just the lines recording the amount of time it took a particular filter or filter group to run

Reply:
I think that would be useful

Reply:
@marip8 I've added timing messages to the filter group, please take a look.